data/hai/hai reveal 4 middle.txt

```diff
@@ -1417,25 +1417,59 @@ mission "Hai Reveal [A14] A Time of Trouble"
 		log `The Hai have threatened the Republic if suitable action isn't taken. Required to escort civilians out of the possible danger zone.`
 		conversation
 			`Upon docking at <planet>, now bereft of civilians, you and Sayari are kept out of things for a little while, as some business occurs behind a Navy security barricade in the direction of where Teeneep's ship docked. Neither Terry nor Turner are are allowed through either, as Navy personnel progressively establish operations and set secure areas on the station. Eventually Batu is sent out to get all of you and escort you in.`
```

Comment:
'Upon docking at <planet>' should be 'Upon docking at <origin>'; playthrough had me confused when I landed on Mountaintop and the conversation said I was on Sunracer. (Sorry I don't know how to contribute changes yet)

Reply:
When you're commenting, you can click the ± inside a page to get the line you want to change. I'll do this one for you. 
```suggestion
			`Upon docking at <origin>, now bereft of civilians, you and Sayari are kept out of things for a little while, as some business occurs behind a Navy security barricade in the direction of where Teeneep's ship docked. Neither Terry nor Turner are are allowed through either, as Navy personnel progressively establish operations and set secure areas on the station. Eventually Batu is sent out to get all of you and escort you in.`
```
I am pretty sure this was changed earlier in another PR.

Reply:
That does sound a bit off. Yes, I will fix that. Thank you for noticing.

Reply:
> I am pretty sure this was changed earlier in another PR.

I've been merging master to this branch frequently, but I have so many text changes that some fixes may have gotten lost in the conflict resolution.

Reply:
I found another planet-origin mixup in part 1, Hai Tour [5] Tour Ends, which I've fixed in #7566. Thanks for getting me to check on those.